src/services/__tests__/classService.test.ts

```
@@ -0,0 +1,629 @@
+          ...createMockClass({ teacher_id: teacher.id }),
+          class_teachers: [{ user_id: teacher.id, role: 'primary', profiles: { full_name: teacher.full_name } }],
+          class_students: [],
+          class_subjects: []
+        }
+      ];
+
+      const classesQuery = {
+        select: jest.fn().mockReturnThis(),
+        eq: jest.fn().mockReturnThis(),
+        order: jest.fn().mockReturnThis(),
+        then: jest.fn().mockResolvedValue({
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)